examples/booking_app/lib/lib_router/routes/hotel_details_routes.dart

```diff
@@ -0,0 +1,26 @@
+part of '../router.dart';
```

Comment:
They are just two routes. Do you think is hood idea to group be into the file routes/routes.dart ?

Reply:
Done